<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExtruderDock</class>
 <widget class="QDockWidget" name="ExtruderDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Extruder Dock</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QLabel" name="extr0_img">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>181</width>
      <height>271</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../img/extruder_v1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="extr1_img">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>60</y>
      <width>181</width>
      <height>271</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../img/extruder_v1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="extr0_title">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 75 24pt &quot;MS Shell Dlg 2&quot;;
color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Extruder 1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="extr0_fan">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>172</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="suffix">
     <string>%</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="extr0_pel">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>210</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="suffix">
     <string>%</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="extr0_heater">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>278</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="suffix">
     <string>°c</string>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
   </widget>
   <widget class="QLabel" name="extr0_clock_label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>85</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Filament</string>
    </property>
   </widget>
   <widget class="QLabel" name="extr0_cold_top">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>temp</string>
    </property>
   </widget>
   <widget class="QLabel" name="extr0_cold_bot">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>245</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>temp</string>
    </property>
   </widget>
   <widget class="QLabel" name="extr0_hotend">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>290</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>temp</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="extr1_pel">
    <property name="geometry">
     <rect>
      <x>441</x>
      <y>210</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="suffix">
     <string>%</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="extr1_cold_top">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>110</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>temp</string>
    </property>
   </widget>
   <widget class="QLabel" name="extr1_clock_label">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>85</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Filament</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="extr1_heater">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>278</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="suffix">
     <string>°c</string>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="extr1_fan">
    <property name="geometry">
     <rect>
      <x>441</x>
      <y>172</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="suffix">
     <string>%</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="extr1_hotend">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>290</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>temp</string>
    </property>
   </widget>
   <widget class="QLabel" name="extr1_cold_bot">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>245</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>temp</string>
    </property>
   </widget>
   <widget class="QLabel" name="extr1_title">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 75 24pt &quot;MS Shell Dlg 2&quot;;
color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Extruder 2&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>16</width>
      <height>341</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="extr0_coldextr">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>350</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Cold Extrusion</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="extr1_coldextr">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>350</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Cold Extrusion</string>
    </property>
   </widget>
   <widget class="Led" name="extr0_filament">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>85</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>15</width>
      <height>15</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>15</width>
      <height>15</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../ressources.qrc">:/img/img/led_off_red.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Led" name="extr1_filament">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>85</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>15</width>
      <height>15</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>15</width>
      <height>15</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../ressources.qrc">:/img/img/led_off_red.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Led</class>
   <extends>QLabel</extends>
   <header>Led.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
